docs/installation.md

```diff
@@ -13,7 +13,7 @@ brew install apify-cli
 
 ## Via NPM
 
-First, make sure you have [Node.js](https://nodejs.org) version 16 or higher with NPM installed on your computer:
+First, make sure you have [Node.js](https://nodejs.org) version 18 or higher with NPM installed on your computer:
 
 ```bash
```

Comment:
```suggestion
```bash showLineNumbers
```

Reply:
In new styleguide released ~week ago I mentioned that we should add this to every code examples even when they are 1 line long, both for consistency and for QoL for users